docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -454,6 +454,26 @@ In the table above, whenever `NotificationResponseReceivedListener` is triggered
 
 > **info** When the app is not running or killed and is restarted by tapping on a notification, the `NotificationResponseReceivedListener` may or may not be triggered. To reliably capture the response, we recommend using `useLastNotificationResponse` or `getLastNotificationResponseAsync`.
 
+### Background notification configuration&ensp;<PlatformTags platforms={["ios"]} />
+
+To be able to run background notifications on iOS, you need to add the `remote-notification` value to the `UIBackgroundModes` array in your app's **Info.plist** file.
+
+**If you're using [CNG](/workflow/continuous-native-generation/)**, the required `UIBackgroundModes` configuration will be applied automatically by prebuild.
```

Comment:
@chrfalch 

the next paragraph "Background notifications" previously contained this:

```
- In the array under the `ios.infoPlist.UIBackgroundModes` key in your [app config](/workflow/configuration/), add the values `remote-notification` and `processing`.
```

I believe that instruction is still valuable (?) - or maybe I don't understand what `If you're using, the required `UIBackgroundModes` configuration will be applied automatically by prebuild.` really means. How does this "automatic" configuration work?

Reply:
Hi! Yes, the line was fixed since It contained an error - `processing` has nothing to do with notifications - it is for the new `BGTaskScheduler` API (which could be used in a new `expo-background-task` package probably). 

"Automatic" means that when you're using CNG (continuous native generation) `eas:build` or `prebuild` will update your info.plist file for you - there are plugins in each of the library responsible for doing so. I had a few discussions with Brent about how we could "standardize" on a specific wording in our docs - which resulted in this.

Reply:
then I believe (?) that the following should stay:

`- In the array under the `ios.infoPlist.UIBackgroundModes` key in your [app config](/workflow/configuration/), add the value `remote-notification``

What confuses me is that after reading the docs I feel I don't need to do _anything at all_ to set up background notifications on iOS if I'm using prebuild.

However, we can't / shouldn't automatically decide for users whether to add `remote-notification` to `UIBackgroundModes` - I believe the users should do that themselves.

The [config plugin for notifications](https://github.com/expo/expo/blob/main/packages/expo-notifications/plugin/src/withNotifications.ts#L8) does not add `remote-notification` to `UIBackgroundModes` this and neither does [the one in task manager](https://github.com/expo/expo/blob/db38d257a2aaacc8562f553e72f07d00bcb5538f/packages/expo-task-manager/plugin/src/withTaskManager.ts#L5) afaict.